specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/disk.json

```diff
@@ -820,6 +829,15 @@
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
         },
+        "burstingEnabledTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The most recent time when bursting was enabled."
+        },
+        "readOnly": {
+          "type": "boolean",
+          "description": "A property to indicate whether the disk is ReadOnly. At this time, this is only supported for Direct Drive Disks."
```

Comment:
Check with Cosmin if this only applies to shared disks. Instead of direct drive say "Ultra_LRS or PremiumV2_LRS"